### ISSUE TITLE
Unable to retrieve pull secret openshift-operator-lifecycle-manager/coreos-pull-secret for openshift-operator-lifecycle-manager/olm-operator...

### ISSUE THREAD

**Issue by jlebon** [operator-framework/operator-lifecycle-manager#607]
Seeing this spammed over and over in my master logs on a fresh 4.0 install:

```
Dec 06 13:57:29 test1-master-0 hyperkube[2893]: W1206 13:57:29.875130    2893 kubelet_pods.go:841] Unable to retrieve pull secret openshift-operator-lifecycle-manager/coreos-pull-secret for openshift-operator-lifecycle-manager/olm-operator-68957f688b-59nsx due to secrets "coreos-pull-secret" not found.  The image pull may not succeed.
Dec 06 13:57:36 test1-master-0 hyperkube[2893]: W1206 13:57:36.375690    2893 kubelet_pods.go:841] Unable to retrieve pull secret openshift-operator-lifecycle-manager/coreos-pull-secret for openshift-operator-lifecycle-manager/catalog-operator-5c75f9c68d-r52rx due to secrets "coreos-pull-secret" not found.  The image pull may not succeed.
...
```

**Comment by wking** [comment]
The installer has been pushing `coreos-pull-secret` into the `kube-system` namespace (not the `openshift-operator-lifecycle-manager` namespace) since at least [here][1] ([landed 2018-09-20][2]).  I [saw these same errors][3] in a run that included:

```console
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.0.0-4 --commits  | grep operator-framework
  olm    https://github.com/operator-framework/operator-lifecycle-manager    8429cb39b98e644318b8fae52406197cf4ab35a1
```

#584 and #591 have landed since 8429cb39b9, but nothing there is jumping out at me as fixing this issue.

[1]: https://github.com/openshift/installer/commit/4d636d346babdf32f5d86ed34d0a084926016be9#diff-8fd5416bccb910470b033b137fe6dbb6R15
[2]: https://github.com/openshift/installer/pull/286#event-1858630660
[3]: https://github.com/openshift/installer/pull/848#issuecomment-445515270

**Comment by ecordell** [comment]
Thanks for the bug report - you can safely ignore these errors, but we will update to remove them.

Older versions of OLM shipped catalogs that required a pull secret which we expected to exist in the same namespace. But none of the current operators require that secret anymore. The logs are annoying but have no effect on OLM or the operators it installs today.

**Comment by ecordell** [comment]
Fixed in #632 #636
